categories for populating schedule added

TereseBo opened the pull request:
Adde custom posttype for holding schedule entries.
Added categories for specifying schedulecontents

Darkshiira commented:
Found a bug, if you don't schedule a subject/time or weekday, the entire site will error. :D
But other than that, looks really good!

TereseBo commented:
Replaced query with loop querying the weekdays.
Added sorting by schedule_starttime.
Hav no idea how to add spacers when there are slots whitout classes in the schedule
